daemon/entrypoint.sh

```diff
@@ -226,6 +227,9 @@ function start_osd {
       activate)
          osd_activate
          ;;
+      devices)
```

Comment:
No real problem with it.  We just already have several, and their names are already somewhat nebulous and difficult for newcomers to fathom.